### PR TITLE
Laravel 11

### PR DESCRIPTION
**Pull request by robsontenorio**

### CODE REVIEW

**Comment by driesvints** [comment]
Heya. Can you also add the new test bench version and update the matrix build on the GHA?

**Comment by robsontenorio** [comment]
Done! But it fails because v11 dependencies  :(

File: composer.json [hunk 1]
```diff
@@ -26,10 +26,10 @@
...
-        "illuminate/contracts": "^8.0|^9.0|^10.0",
-        "illuminate/filesystem": "^8.0|^9.0|^10.0",
-        "illuminate/support": "^8.0|^9.0|^10.0",
-        "illuminate/view": "^8.0|^9.0|^10.0",
+        "illuminate/contracts": "^8.0|^9.0|^10.0|^11.0",
+        "illuminate/filesystem": "^8.0|^9.0|^10.0|^11.0",
+        "illuminate/support": "^8.0|^9.0|^10.0|^11.0",
+        "illuminate/view": "^8.0|^9.0|^10.0|^11.0",
         "symfony/console": "^5.3|^6.0",
         "symfony/finder": "^5.3|^6.0"
```
Comment (driesvints) [inline composer.json]: You'll have to add Symfony v7 constraints here as well.

**Review by driesvints — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (robsontenorio) [reply composer.json]: Did it! . But, the problem now is `illuminate/contracts`  windows L11.  Should I remove it?


image

**Comment by driesvints** [comment]
What's weird is that the exact same build on Ubuntu does work. I'm gonna leave this open until after Laravel v11's release and then merge and tag.

**Comment by driesvints** [comment]
Is disabled the Laravel v11 windows build for now. Thanks!

**Comment by owenvoke** [comment]
This might be happening because Windows treats `^` as an escape character. 🤔 If this is the issue, an alternative is to change to `"illuminate/contracts=${{ matrix.laravel }}.*"` 🙂 I would have thought it'd have been fine though, as it's quoted, but perhaps not.

**Comment by driesvints** [comment]
@owenvoke seems that worked! https://github.com/blade-ui-kit/blade-icons/pull/224

Thanks :)

**Pull request merged**